client/a8c-for-agencies/sections/overview/overview.tsx

```diff
@@ -0,0 +1,25 @@
+import { useTranslate } from 'i18n-calypso';
+import Layout from 'calypso/a8c-for-agencies/components/layout';
+import LayoutBody from 'calypso/a8c-for-agencies/components/layout/body';
+import LayoutHeader, {
+	LayoutHeaderTitle as Title,
+} from 'calypso/a8c-for-agencies/components/layout/header';
+import LayoutTop from 'calypso/a8c-for-agencies/components/layout/top';
+import MobileSidebarNavigation from 'calypso/a8c-for-agencies/components/sidebar/mobile-sidebar-navigation';
+
+export default function Overview() {
+	const translate = useTranslate();
+
+	return (
+		<Layout title="Overview" wide sidebarNavigation={ <MobileSidebarNavigation /> }>
```

Comment:
I think we can now update the title to `translate( 'Agency HQ Overview' )`

```
const title = translate( 'Agency HQ Overview' )
```

```suggestion
		<Layout title={ title } wide sidebarNavigation={ <MobileSidebarNavigation /> }>
```

Reply:
Added in 721466afc10e6f3d1a5b58265fb3a550c440f832 :+1: